test/e2e/scheduling/preemption.go

```diff
@@ -460,7 +461,12 @@ var _ = SIGDescribe("PreemptionExecutionPath", func() {
 		}
 	})
 
-	ginkgo.It("runs ReplicaSets to verify preemption running path", func() {
+	/*
+	   Release : v1.16
+	   Testname: Verify preemption of pod ReplicaSets
+	   Description: MUST create pods with ReplicaSets and verify pod's replica count
+	*/
+	framework.ConformanceIt("runs ReplicaSets to verify preemption running path", func() {
```

Comment:
I think the description needs some work to better match what the test actually does.

Reply:
This is a test that is specifically designed to check race conditions in preemption. As far as conformance is concerned, I would prefer to make the test at line 95 a conformance test instead of this one.